src/snowflake/telemetry/_internal/opentelemetry/exporter/otlp/proto/common/_log_encoder.py

```diff
@@ -0,0 +1,12 @@
+from typing import Sequence
+
+from opentelemetry.exporter.otlp.proto.http._log_exporter.encoder import _ProtobufEncoder
+from opentelemetry.proto.collector.logs.v1.logs_service_pb2 import ExportLogsServiceRequest
+from opentelemetry.sdk._logs import LogData
+
+
+def encode_logs(batch: Sequence[LogData]) -> ExportLogsServiceRequest:
+    return _ProtobufEncoder.encode(batch)
```

Comment:
We discussed to not have this for the moment no?

Reply:
It was already implemented before that discussion. For now I just matched what's in latest opentelemetry-python's opentelemetry-exporter-otlp-proto-common module (pasted below for reference). Since we're not ready to support encode_logs with this method signature, I'll change this to what I believe the long term support method signature will look like, but with an empty implementation for now.

```python
# opentelemetry-python/exporter/opentelemetry-exporter-otlp-proto-common/src/opentelemetry/exporter/otlp/proto/common/_internal/_log_encoder/__init__.py
def encode_logs(batch: Sequence[LogData]) -> ExportLogsServiceRequest:

# opentelemetry-python/exporter/opentelemetry-exporter-otlp-proto-common/src/opentelemetry/exporter/otlp/proto/common/_internal/metrics_encoder/__init__.py
def encode_metrics(data: MetricsData) -> ExportMetricsServiceRequest:

# opentelemetry-python/exporter/opentelemetry-exporter-otlp-proto-common/src/opentelemetry/exporter/otlp/proto/common/_internal/trace_encoder/__init__.py
def encode_spans(
    sdk_spans: Sequence[ReadableSpan],
) -> PB2ExportTraceServiceRequest:
```

Reply:
I've added a proposed method signature that relies directly on Python's built-in LogRecord for the incoming data. Please take a look.